doc/source/install.rst

```diff
@@ -47,7 +28,7 @@ you haven't created and activated one already, you can by running:
 
 .. code-block:: bash
 
-    $ conda create -n my_satpy_env python
+    $ conda create -c conda-forge -n my_satpy_env python
```

Comment:
Would it be better to advise to install `satpy` directly at env creation to make the installation faster? Or add a short comment like "It is also possible to define all the wanted packages in the environment creation command, which makes the installation faster and might solve some dependency issues between package versions."

Reply:
Yeah good point. I'll see if I can restructure this.